db/seeds_for_by_election.rb

```diff
@@ -18,7 +18,7 @@ class SeedHelper
 puts "\nParties"
 
 Db::Fixtures::Be2022::Party.all.each do |party|
-  ::Party.find_or_create_by(name: party[:name], color: party[:colour])
+  ::Party.find_or_create_by(name: party[:name], shortName: party[:shortName], color: party[:colour])
```

Comment:
I **think** this is the right thing to do? I don't know anything about how Ruby does databases.